src/components/Colors/Colors.stories.tsx

```diff
@@ -0,0 +1,254 @@
+import * as React from "react";
+
+import { boolean, text } from "@storybook/addon-knobs";
+import cssVariables from "../../helpers/CSSVariablesHelper";
+import getCSSVariable from "../../helpers/getCSSVariable";
+import UIDocCard from "./UIDocCard";
+import Card from "../Card/Card";
+import List from "../List/List";
+import Heading from "../Heading/Heading";
+import { ListTypes } from "../List/ListTypes";
+
+export default {
+    title: "Colors",
+    decorators: [(storyFn) => <div style={{ padding: "2%" }}>{storyFn()}</div>],
+};
+
+const uiVariables: { [k: string]: string } = {};
+const grayScaleVariables: { [k: string]: string } = {};
+
+for (const [key, value] of Object.entries(cssVariables)) {
+    if (key.startsWith(" --ui-gray")) {
+        grayScaleVariables[key] = value;
+    } else if (key.startsWith(" --ui")) {
+        uiVariables[key] = value;
+    }
+}
+
+let uiVariableDocs = [];
+let grayscaleDocs = [];
+
+for (const [key, value] of Object.entries(uiVariables)) {
+    uiVariableDocs.push(
+        <li style={{ textAlign: "center" }}>
+            <UIDocCard backgroundColor={value}>
+                {key}: {value}
+            </UIDocCard>
+        </li>
+    );
+}
+
+for (const [key, value] of Object.entries(grayScaleVariables)) {
+    grayscaleDocs.push(
+        <li style={{ textAlign: "center" }}>
+            <UIDocCard backgroundColor={value}>
+                {key}: {value}
+            </UIDocCard>
+        </li>
+    );
+}
```

Comment:
Not necessary for this PR, but the code for these two loops are very similar. You can make this into a function where you pass in the object and the array (ie `makeUIDocCard(grayScaleVariables, grayscaleDocs)`).